docs/src/troubleshooting.md

```diff
@@ -45,6 +45,80 @@ It's known that Playwright is not working as expected when some Microsoft Edge /
 - `UserDataDir` - This policy is used to specify the location of the user data directory. Playwright uses a temporary directory for user data, so this policy is not compatible with Playwright. See discussion in [this bug](https://github.com/microsoft/playwright/issues/17448).
 - `ExtensionInstallForcelist` - This policy is used to specify a list of extensions that should be installed. Playwright's browser close will not work if this policy is set. See discussion in [this bug](https://github.com/microsoft/playwright/issues/17299).
 
+## Error codes
+
+### Browser management errors
```

Comment:
consider introducing playwright error classes to simplify catching internal errors (a `BrowserError`, `PlaywrightError` and more)